### PR TITLE
Add support for radio7.lv

### LINKED ISSUE #2542: Radio7
I am looking for an option to scrobble my favorite radio http://radio7.lv to http://last.fm
. unfortunately it seems that extension does not support this page (sadly) This radio has only music and you can always see artist and song name playing now. Any chance to scrobble anything from  http://radio7.lv ? 
thanks

### PR DESCRIPTION
**Pull request by inverse**
Resolves #2542

### CODE REVIEW

File: src/connectors/radio7.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+Connector.playerSelector = '#page-wrap';
+
+Connector.artistTrackSelector = '#now';
+
+Connector.isPlaying = () => Util.isElementVisible('#jp_container_1 .jp-pause');
```
Comment (alexesprit) [inline src/connectors/radio7.js:7]: It's better to use `Connector.pauseButtonSelector`.

**Review by alexesprit — CHANGES_REQUESTED** [review]

File: src/connectors/radio7.js (hunk 1 above)
Comment (inverse) [reply src/connectors/radio7.js]: Crazy - I was trying with this yesterday and it wasn't working :D

**Comment by alexesprit** [comment]
The "Changes requested" status cannot be clear after the merge. :(

**Comment by Edislv** [comment]
Any chance to scrobble?

**Comment by alexesprit** [comment]
I'll publish a new version on the weekend.

**Comment by Edislv** [comment]
Hi everyone! Many thanks for fixing this 2 tears ago - you amazing! 
Seems that page radio7.lv updated hypertext, so can't copy paste it. Is still possible scrobble this? or in that configuration it is not recognizable?

**Comment by inverse** [comment]
Hi @Edislv!

I fixed this again in https://github.com/web-scrobbler/web-scrobbler/pull/3232 which will be in the next release.

Please create an issue again next time to make it easier for us to track 🙏🏼

**Pull request merged**